templates/home_page.hbs

```diff
@@ -1,72 +1,21 @@
 <section class="section hero">
-  <div class="hero-inner">
-    {{search submit=false instant=settings.instant_search class='search search-full'}}
-  </div>
-</section>
-
-<div class="container">
-  <section class="section knowledge-base">
-    <section class="categories blocks">
-      <ul class="blocks-list">
-        {{#each categories}}
-          {{#if ../has_multiple_categories}}
-            <li class="blocks-item">
-              <a href='{{url}}' class="blocks-item-link">
-                <span class="blocks-item-title">{{name}}</span>
-                <span class="blocks-item-description">{{excerpt description}}</span>
-              </a>
-            </li>
-          {{else}}
-            {{#each sections}}
-              <li class="blocks-item">
-                <a href='{{url}}' class="blocks-item-link">
-                  <span class="blocks-item-title">
-                    {{name}}
-                  </span>
-                  <span class="blocks-item-description">{{excerpt description}}</span>
-                </a>
-              </li>
-            {{/each}}
-          {{/if}}
+  <div class="container hero-inner">
+    <h1>How can we help?</h1>
+    {{search submit=false placeholder='Search Help Center' instant=settings.instant_search class='search search-full'}}
+    {{#if promoted_articles}}
+      <ul class="hero__popular-topics" role="list">
+        <li class="hero__popular-topics--item" role="listitem">Popular Topics:</li>
```

Comment:
for bem `--` is a modifier. To follow proper bem, here we would something like `hero__topic-item`